models/database.go

```diff
@@ -235,6 +235,12 @@ var databaseSchema = [][]string{
 			AND settings->'metrics_resolutions'->>'mr' = '5000000000'
 			AND settings->'metrics_resolutions'->>'lr' = '60000000000';`,
 	},
+
+	11: {
+		`ALTER TABLE agents
+			ADD COLUMN rds_basic_metrics_disabled BOOLEAN NOT NULL DEFAULT FALSE,
+			ADD COLUMN rds_enhanced_metrics_disabled BOOLEAN NOT NULL DEFAULT FALSE`,
```

Comment:
Please drop default in the next query. See migrations 3, 4 and 8 for examples.

@BupycHuk I think we should add a comment about that somewhere. Above migrations in this file?
In a nutshell, the problem is that Go's zero values and non-zero default values in database do play nicely together in INSERTs and UPDATEs. As a general rule, we should avoid default values in schema definition.

Reply:
Fixed.